src/platform_impl/web/event_loop/mod.rs

```diff
@@ -31,7 +31,7 @@ impl<T> EventLoop<T> {
 
     pub fn run<F>(self, event_handler: F) -> !
     where
-        F: 'static + FnMut(Event<T>, &RootEventLoopWindowTarget<T>, &mut ControlFlow),
```

Comment:
I think this would be unsound on Web, since `run` technically returns immediately, meaning that any references held in this closure would be left dangling.

Reply:
On Web `run()` doesn't return immediately, it interrupts the function without unwinding, so the closure passed into `run()` leaks and therefore this should work just fine.